service/server.go

```diff
@@ -55,6 +55,8 @@ type ServerStaticConf struct {
 	DeprecationWarning string
 	// GitCommit is the git commit from which the server was built.
 	GitCommit string
+	// HTTPTimeout is the timeout of the S3 clients' http.Client
```

Comment:
also the custom http client. I'd probably say something more general like 'server timeout'